cmd/cluster/kubevirt/create.go

```diff
@@ -0,0 +1,76 @@
+package kubevirt
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	apifixtures "github.com/openshift/hypershift/api/fixtures"
+	"github.com/openshift/hypershift/cmd/cluster/core"
+	"github.com/spf13/cobra"
+	utilrand "k8s.io/apimachinery/pkg/util/rand"
+)
+
+func NewCreateCommand(opts *core.CreateOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "kubevirt",
+		Short:        "Creates basic functional HostedCluster resources for KubeVirt platform",
+		SilenceUsage: true,
+	}
+
+	opts.KubevirtPlatform = core.KubevirtPlatformCreateOptions{
+		Memory: "4Gi",
+		CPU:    2,
+		Image:  "",
+	}
+
+	cmd.Flags().StringVar(&opts.KubevirtPlatform.Memory, "kubevirt-memory", opts.KubevirtPlatform.Memory, "In KubeVirt platform - the amount of memory which is visible inside the Guest OS (type BinarySI, e.g. 5Gi, 100Mi)")
```

Comment:
How about `node-memory` or just `memory`? We are inside the `kubevirt` subcommand, so we do not in general need a `kubevirt-` prefix.

Reply:
Changed to memory